packages/docusaurus-theme-classic/src/theme/Admonition/index.tsx

```diff
@@ -69,41 +71,82 @@ function CautionIcon() {
 type AdmonitionConfig = {
   iconComponent: React.ComponentType;
   infimaClassName: string;
+  label: ReactNode;
 };
 
-const AdmonitionConfigs: {[key: string]: AdmonitionConfig | string} = {
+// eslint-disable-next-line @typescript-eslint/consistent-indexed-object-style
+const AdmonitionConfigs: Record<Props['type'], AdmonitionConfig> = {
   note: {
     infimaClassName: 'secondary',
     iconComponent: NoteIcon,
+    label: (
+      <Translate
+        id="theme.admonition.note"
+        description="The default label used for the Note admonition (:::note)">
+        note
+      </Translate>
+    ),
   },
   tip: {
     infimaClassName: 'success',
     iconComponent: TipIcon,
+    label: (
+      <Translate
+        id="theme.admonition.tip"
+        description="The default label used for the Tip admonition (:::tip)">
+        tip
+      </Translate>
+    ),
   },
   danger: {
     infimaClassName: 'danger',
     iconComponent: DangerIcon,
+    label: (
+      <Translate
+        id="theme.admonition.danger"
+        description="The default label used for the Danger admonition (:::danger)">
+        danger
+      </Translate>
+    ),
   },
   info: {
     infimaClassName: 'info',
     iconComponent: InfoIcon,
+    label: (
+      <Translate
+        id="theme.admonition.info"
+        description="The default label used for the Info admonition (:::info)">
+        info
+      </Translate>
+    ),
   },
   caution: {
     infimaClassName: 'warning',
     iconComponent: CautionIcon,
+    label: (
+      <Translate
+        id="theme.admonition.caution"
+        description="The default label used for the Caution admonition (:::caution)">
+        caution
+      </Translate>
+    ),
   },
+};
+
+// Legacy aliases, undocumented but kept for retro-compatibility
+const aliases = {
   secondary: 'note',
   important: 'info',
   success: 'tip',
   warning: 'danger',
```

Comment:
"Warning" is used more often than "danger", at least for our docs. Do we really want to not translate it?

Reply:
good point, this is a bit annoying to have these aliases and would require duplicating some translation duplication

btw I'm not fan at all of "caution". "warning" is more natural to me.

Do we really need distinct/different labels for both cases?

Reply:
@forresst do you find it useful to have 2 orange admonitions for warning/caution ?

They are currently translated in French as "avertissement" / "attention", but to me we could as well use "attention" for both and it would remain a good translation

Reply:
@slorber You are right. I agree with you. One is enough. "attention" is perfect

Reply:
"Warning" is the red one, "caution" is the orange one—that's the confusing part.

Reply:
@forresst personally I find "Attention" more usual in French and would use it in both cases

I guess all this is quite subjective 🤪 hopefully it is now customizable

Reply:
oh yes, I'm misunderstanding things now.

Our alias "warning" -> "danger" feels really wrong to me. To me a warning should be yellow/orange.

BTW it looks like I'm not alone to think so

https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

https://nextra.vercel.app/themes/docs/callout

https://vitepress.vuejs.org/guide/markdown.html#custom-containers

https://docs.gitbook.com/editing-content/rich-content/with-command-palette#hints-and-callouts

I'll open a dedicated issue, we should probably do like all others, but we need a way to migrate this gracefully or this will be annoying for our users

Reply:
That's very right. It has always been tricky for me as well. A breaking change is probably worth it, considering `:::warning` wasn't documented anyway...

Reply:
https://github.com/facebook/docusaurus/issues/7558